src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,27 @@
+// TEXT
+$medium-size-text: 12px;
+$small-size-text: 10px;
+
+//WEIGHT
+$normal-weight: 500;
+$bold-weight: 700;
+
+// COLOR
+$emphasized-color: #060b35;
+$lightened-color: #616070;
+$white-color: #fff;
+$black-color: #000;
+$card-border-color: #f3f3f3;
+$button-color: #00acdc;
+$box-color: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+// LOGO, STARS SIZES
+$logo-size: 40px;
+$star-size: 16px;
+
+// CARDS SETTINGS
+$margin-bottom: 16px;
+$card-width: 200px;
+$cards-right-gap: 48px;
+$cards-top-gap: 46px;
+$catalog-width: 4 * $card-width + $cards-right-gap * 3;
```

Comment:
Would be nice to create variable for cards amount, because now it looks like magic number

Reply:
@OleksiiBlezniuk you mean, I should create a variable named, for example, "$cards-in-row: 4;"?

Reply:
Yep, exactly